glean-core/ios/Glean/Glean.swift

```diff
@@ -232,7 +232,7 @@ public class Glean {
     ///
     /// If the ping currently contains no content, it will not be assembled and
     /// queued for sending.
-    func submitPingsByName(pingNames: [String]) {
+    func submitPingsByName(pingNames: [String], reason _: String? = nil) {
```

Comment:
Are you sure you meant to have `reason _: String? = nil`?  This means that if someone passes a `reason` to the function, that the function should ignore it and basically doesn't provide access to it inside of the function.

Simply having the default value will allow you to omit it if there is no reason.
```suggestion
    func submitPingsByName(pingNames: [String], reason: String? = nil) {
```

Reply:
Weird -- the Swift formatter told me to add it (and failed without it).  But I agree, that doesn't make much sense.

Reply:
Since `reason` isn't used in the function, the formatter thought it was being helpful by suggesting you make it explicit that you weren't using it.

Reply:
I'm guessing it was supposed to be passed to the other function?